ravenframework/raven_ec_slurm_command.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if test -n "$SLURM_SUBMIT_DIR"; then
+    echo Moving to working dir: ${SLURM_SUBMIT_DIR}
+    cd $SLURM_SUBMIT_DIR
+fi
+
+source /etc/profile.d/modules.sh
```

Comment:
I saw you made modifications in ec_qsub_command to include an if else check, should we do the similar thing here?

Reply:
For INL clusters, all the slurm ones use /etc/profile.d/modules.sh, but since may be used as a template for other clusters I changed it to use the if check.